docs/ProjectPlanning.md

```diff
@@ -112,9 +132,9 @@ Least constraints here- it will serve as an API endpoint for clients and admin i
 - Classroom: single room at a school where groups of students and teachers participate in class sessions (poorly worded)
 - Teacher: Person who teaches and takes attendance
 - Student: Person who attends classes and receives lessons
-- Subject: A lesson plan that us used to teach a class
-- Class: A schedule of lessons covering a subject given by a specific teacher to a prescribed group of students
-- Lesson: A single scheduled session of a class
+- Subject: A lesson plan that us used to teach a class (does this matter for attendance keeping purposes?)
```

Comment:
This really depends on what the final schema looks like. The data model that I wrote down is more of a brainstorm than something fixed.